client/src/unifycr-defs.h

```diff
@@ -1,8 +1,15 @@
+/* holds debug level for unifycr, this defaults to zero
+ * in unifycr_init, but can be changed at runtime by setting
+ * the DEBUG env variable to a value greater than 1
```

Comment:
I'd prefer to name it `UNIFYCR_DEBUG` for consistency with other unifycr environment variables and  to reduce likelihood of name collision.